src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -1087,6 +1087,9 @@ def certificate_authority_manager(self) -> chip.CertificateAuthority.Certificate
     def dut_node_id(self) -> int:
         return self.matter_test_config.dut_node_ids[0]
 
+    def get_endpoint(self, default: Optional[int] = 0) -> int:
```

Comment:
opinion question - do you think the default makes sense here or do you think it makes more sense to throw an error if the test uses an endpoint, but one isn't supplied. 

@raju-apple - thoughts? It might be a bit disruptive, but possibly less error prone?

Either way, let's not block this PR on that.

Reply:
I think we should just mandate an endpoint id to be provided, we can do that as a follow up if we do not want to make that change here.

Reply:
follow up issue - https://github.com/project-chip/matter-test-scripts/issues/450